--- hwinfo-13.11/src/hd/Makefile.orig	2005-07-06 12:50:26.000000000 +0200
+++ hwinfo-13.11/src/hd/Makefile	2007-05-23 22:01:21.136191593 +0200
@@ -4,7 +4,7 @@
 
 include $(TOPDIR)/Makefile.common
 
-CFLAGS		+= -I /usr/include/dbus-1.0 -I /usr/lib64/dbus-1.0/include -I /usr/lib/dbus-1.0/include
+CFLAGS		+= `pkg-config --cflags dbus-1`
 
 version.h: $(TOPDIR)/VERSION
 	@echo "#define HD_VERSION_STRING \"`cat $(TOPDIR)/VERSION`\"" >$@
